README.rst

```diff
@@ -33,9 +33,17 @@ Unit tests are also run on CI.
 
 The system tests are run using this command::
 
-  $ nix-build nixos/system-tests.nix
+  $ sudo --preserve-env nix-build nixos/system-tests.nix
```

Comment:
Yes you are! At least I needed that. Didn't you @exarkun ?

Reply:
Indeed, I don't need sudo to run this but I'm not sure why. :/  It might be because of one of the groups my user account belongs to - eg I'm in wheel and disk (neither of which seems quite believable as an explanation...).  Unfortunately it's not quite obvious to me exactly what the minimal required permissions for running QEMU are and I find the QEMU docs to be pretty opaque.

We could add another note here suggesting that what you really need is the ability to run a QEMU VM and that `sudo --preserve-env` may be one straightforward way to obtain this (but leave people to pick a better way to achieve that goal if they know of one).

Reply:
Turns out privileges aren't needed after all. Sorry for the confusion. I corrected the docs in 415c7d45289526ca4568e2e79eb76e5d8da62d3f. (Using the semantic linefeeds already paid off!) 

This whole misunderstanding seems to have stemmed from my trying to fix things since I assumed VMs would need privileges, and then having run `sudo -E nix-build` put a ton of files & folders into my personal `/home/flo/.cache/nix` folder that belonged to root. `flo` can't overwrite those, obviously, so that is where my "permission problems" came from. D'uh.